src/main/java/Main.java

```diff
@@ -0,0 +1,53 @@
+import figure.*;
```

Comment:
it's a bad practice to use imports with *

Reply:
ok, understood